javascript/src/URDFLoader.d.ts

```diff
@@ -2,7 +2,7 @@ import { LoadingManager, Object3D } from 'three';
 import { URDFRobot } from './URDFClasses';
 
 interface MeshLoadDoneFunc {
-    (mesh: Object3D): void;
+    (mesh: Object3D, err: any): void;
```

Comment:
Can we make "err" optional and report that it should be of type Error?

```
(mesh: Object3D, err?: Error)
```

Additionally it looks like the docs could be updated to include the error argument, as well:

https://github.com/gkjohnson/urdf-loaders/blob/master/javascript/README.md?plain=1#L147

Reply:
Sure. Given that I hadn't seen the `err` argument in use, I wasn't sure what type it would be. I'll make that change and update the docs.